src/awscr-s3/http.cr

```diff
@@ -16,8 +16,9 @@ module Awscr::S3
     end
 
     def initialize(@signer : Awscr::Signer::Signers::V4,
-                   @region : String = standard_us_region)
-      @http = HTTP::Client.new(host)
+                   @region : String = standard_us_region,
+                   @custom_endpoint : String? = nil)
```

Comment:
I couldn't call it `@endpoint` (like the method) because of https://github.com/crystal-lang/crystal/issues/4332